temba/contacts/management/commands/update_es_with_flows_channels.py

```diff
@@ -0,0 +1,76 @@
+from django.core.management import BaseCommand
+from elasticsearch.helpers import bulk
+
+from temba.contacts.search import elastic
+from temba.contacts.models import Contact
+from temba.orgs.models import Org
+
+raw_query = """
+            SELECT id,
+                (
+                    SELECT array_to_json(array_agg(row_to_json(u)))
+                    FROM (
+                        SELECT scheme,
+                            path,
+                            channels_channel.uuid as channel_uuid,
+                            channels_channel.name as channel_name
+                        FROM contacts_contacturn
+                        LEFT JOIN channels_channel ON contacts_contacturn.channel_id = channels_channel.id
+                        WHERE contact_id = contacts_contact.id
+                    ) u
+                ) as urns_channels,
+                (
+                    SELECT array_to_json(array_agg(row_to_json(f)))
+                    FROM (
+                        SELECT DISTINCT ff.uuid, ff.name
+                        FROM flows_flowrun fr
+                        INNER JOIN flows_flow ff on ff.id = fr.flow_id
+                        WHERE fr.contact_id = contacts_contact.id
+                    ) f
+               ) as flows
+            FROM contacts_contact
+            WHERE contacts_contact.is_active = true AND contacts_contact.org_id = %d AND contacts_contact.id > %d
+            ORDER BY contacts_contact.id
+            LIMIT 500000;
```

Comment:
@ihor-palii - Same question from another file: do you think there's the possibility to have more than 500k on this query's return?

Reply:
Can you please run the black (`black --line-length=119 --target-version=py36 temba`) and flake8 (`flake8 temba/ --ignore=E501,F405,T003,E203,W503`) commands?

Reply:
Updated.